docs/blocks/blocks.md

```diff
@@ -27,26 +27,19 @@ Of course, the above only allows for extremely basic blocks. If you want to add
 Registering a Block
 -------------------
 
-So now that you have this block you've created, it's time to actually put it in the game. Thankfully this is extremely simple to do.
-
-As with most things, blocks can be registered using `GameRegistry.register(...)`. For this method your block must have a "registry name" which is just another way of saying "unique name". The preferred method to set your block's registry name is by calling `setRegistryName` on it inside the call to `register`. For instance, `GameRegistry.register(myBlock.setRegistryName("foo"))`.
+Blocks must be [registered][registering] to function.
 
 !!! important
-
-    There is a very large difference between a block in the world and a block in your inventory. A block in the world is governed by an instance of `Block`, it has a blockstate, a 4-bit meta, and maybe a tile entity. Meanwhile, an item in your inventory is controlled by an instance of `Item`, with a 16-bit damage/meta, and maybe an NBT tag. As a bridge between the different worlds of `Block` and `Item`, we have the class `ItemBlock`. `ItemBlock` is a subclass of `Item` that has a field `block` that holds a reference to the `Block` it represents. `ItemBlock` defines some of the behavior of a "block" as an item, like how a right click places the block. It's possible to have a `Block` without an `ItemBlock`. (E.g. `minecraft:water` is a block, but not an item.)
     
-    When you register a block, you *only* register a block. The block does not automatically have an `ItemBlock`. In order to give your block an `ItemBlock`, you should construct an `ItemBlock` from it and register that as well. The simplest way to do this like this: `GameRegistry.register(new ItemBlock(myBlock).setRegistryName(myBlock.getRegistryName()))`.
-
-!!! note
-
-    When using a simple string, the currently active mod's ID will be added. So if I was doing this from "mymod", the real registry name would be "mymod:foo".
-
-It is also possible to pass a `ResourceLocation` directly into `register`, but this is just convenience for calling `setRegistryName` with the `ResourceLocation` beforehand.
+    There is a very large difference between a block in the world and a block in an inventory. A block in the world is governed by an instance of `Block`, it has a blockstate, a 4-bit meta, and maybe a tile entity. Meanwhile, an item in an inventory is controlled by an instance of `Item`, with a 16-bit damage/meta, and maybe an NBT tag. As a bridge between the different worlds of `Block` and `Item`, there exists the class `ItemBlock`. `ItemBlock` is a subclass of `Item` that has a field `block` that holds a reference to the `Block` it represents. `ItemBlock` defines some of the behavior of a "block" as an item, like how a right click places the block. It's possible to have a `Block` without an `ItemBlock`. (E.g. `minecraft:water` is a block, but not an item.)
```

Comment:
Might as well clean this up if you are changing it anyways.

- "very large difference" sounds clunky, if my English does not betray me differences are big, not large, right? In any case, "very" does not fit here imho, "a big difference" is enough.
- "A block in the world is governed by an instance of `Block`" - No, it's not. If anything this should be `IBlockState`, but even that is not true. Moreover "blockstate" and "4-bit meta" are not parallel, but orthogonal concepts. The blockstate exists and is _serialized_ into the metadata.